static/css/components/header-bar.less

```diff
@@ -441,12 +453,25 @@
 
     .search-bar-submit {
       background: url(/static/images/search-icon.svg) center no-repeat;
-      width: 45px;
+      width: 35px;
       height: 45px;
       border-top-left-radius: .3em;
       border-bottom-left-radius: .3em;
     }
 
+    .vertical-separator {
+      border-left: 1px solid #d3d3d3;
```

Comment:
question, for curiosity, noticed color variables are being used in .less files, ex. `@grey, @baby-green`, but can't find where these colors are being assigned in the repo? 😅

Reply:
They're defined here! https://github.com/internetarchive/openlibrary/blob/master/static/css/less/colors.less